src/HandlerStack.php

```diff
@@ -34,8 +34,6 @@ class HandlerStack
      * @param callable $handler HTTP handler function to use with the stack. If no
```

Comment:
This doc is wrong. Should be `?callable` or `callable|null`. I don't know which style you prefer.

Reply:
Nice catch.. fixed.

Reply:
🚀